lms/static/sass/search/_search.scss

```diff
@@ -0,0 +1,174 @@
+.search-bar,
+.course-index .search-bar,
+.dashboard .search-bar {
+
+  @include box-sizing(border-box);
+  position: relative;
+  padding: 5px;
+  font-family: $sans-serif;
```

Comment:
Shouldn't need to declare font-family here, or anywhere else in this file unless it needs to be overridden. Browser default is inheriting for this property.